test/RSocketTests.h

```diff
@@ -16,21 +15,11 @@ namespace rsocket {
 namespace tests {
 namespace client_server {
 
-// Helps prevent against port collisions.
-inline uint16_t randPort() {
-  std::random_device device;
-  std::uniform_int_distribution<uint16_t> dis(9000, 65000);
-
-  auto const n = dis(device);
-  return static_cast<uint16_t>(n);
-}
-
 inline std::unique_ptr<RSocketServer> makeServer(
-    uint16_t port,
     std::shared_ptr<rsocket::RSocketResponder> responder) {
```

Comment:
can we make the signature as follows?
std::unique_ptr<RSocketServer> makeServer(
      std::shared_ptr<rsocket::RSocketResponder> responder,
      int& usedPort);

so that tests can use the usedPort value to makeClient?